modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -17,21 +17,21 @@ The following `ImageSetConfiguration` file uses a registry storage backend and i
 .Example `ImageSetConfiguration` file
 [source,yaml]
 ----
-apiVersion: mirror.openshift.io/v1alpha1
+apiVersion: mirror.openshift.io/v1alpha2
 kind: ImageSetConfiguration
 archiveSize: 4
 storageConfig:
  registry:
    imageURL: example.com/example/oc-mirror
    skipTLS: false
 mirror:
- ocp:
+ platform:
    channels:
-     - name: stable-4.9
+     - name: stable-4.10
```

Comment:
@jpower432 Should we use stable-4.11 and v4.11 in these examples instead of 4.10?